Makefile

```diff
@@ -81,7 +81,7 @@ IMAGE_NAME=test-infra
 IMAGE_REG_NAME=quay.io/itsoiref/$(IMAGE_NAME)
 
 # oc deploy
-KUBECONFIG := $(or $(KUBECONFIG),${HOME}/.kube/config)
+#KUBECONFIG := $(or $(KUBECONFIG),${HOME}/.kube/config)
```

Comment:
?

Reply:
It defined in the code, no need to set it also here